Results/Ca/ca_0_ca_soma0_0.out

```diff
@@ -0,0 +1,8002 @@
+0 2e-05
```

Comment:
Are all the `.out` files generated data? If so, is there any reason to include them in the source code?

Reply:
Same question for the pdfs under `Figures`

Reply:
For the execution of the NEURON code you will need the directories `Results/Ca` and `Results/FI` and `Figures` but not the figures.

Reply:
Possibly we could add the directories - if they are needed and missing - but skip the .out and .pdf files?

Reply:
Ok, added this in file fig2_validation.py